File: 1-memento/java/example-word-editor-v1/EditorState.java
public class EditorState {
    private final String content;
    private final String fontName;
    private final Integer fontSize;

    public EditorState(String content, String fontName, Integer fontSize){
        this.content = content;
        this.fontName = fontName;
        this.fontSize = fontSize;
    }

    public String getContent(){
        return this.content;
    }

    public String getFontName(){
        return this.fontName;
    }

    public Integer getFontSize(){
        return this.fontSize;
    }
}

File: 1-memento/js/example-word-editor-v1/EditorState.js
module.exports = class EditorState{
    #content;
    #fontName;
    #fontSize;
    
    constructor(content, fontName, fontSize){
        this.#content = content;
        this.#fontName = fontName;
        this.#fontSize = fontSize;
    }
    getContent(){
        return this.#content;
    }
    getFontName(){
        return this.#fontName;
    }
    getFontSize(){
        return this.#fontSize;
    }
}